lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -293,6 +305,9 @@ struct CircuitLoweringState {
   /// If true, non-constant reset values of async reset registers are printed as
   /// errors and cause the pass to abort. Otherwise they are warnings.
   const bool nonConstAsyncResetValueIsError = true;
+
+  // The Dut, if it is found
```

Comment:
```suggestion
  // The design-under-test (DUT), if it is found.  This will be set if a
  // "sifive.enterprise.firrtl.MarkDUTAnnotation" exists.
```